contentcuration/contentcuration/serializers.py

```diff
@@ -618,6 +618,12 @@ class Meta:
         fields = ('email', 'first_name', 'last_name', 'id')
 
 class InvitationSerializer(BulkSerializerMixin, serializers.ModelSerializer):
+    channel_name = serializers.SerializerMethodField('retrieve_channel_name')
```

Comment:
Use this to display the channel name on the invitation without loading everything